components/document/document_view.hpp

```diff
@@ -14,6 +14,7 @@ namespace components::document {
 
         document_view_t();
         explicit document_view_t(document_ptr document);
+        explicit document_view_t(document_view_t&& doc_view);
```

Comment:
```suggestion
        explicit document_view_t(document_view_t&& doc_view) = default;
```
Can we use default move ctor?